docs/thermometer.md

```diff
@@ -0,0 +1,1879 @@
+ strategy:
+   class_name: Strategy
+   args:
+     country: UK
+     max_row_price: 4
+     max_tx_fee: 2000000
+     currency_id: 'FET'
+     ledger_id: 'fetchai'
+     is_ledger_tx: True
+ dialogues:
+   class_name: Dialogues
+   args: {}
+protocols: ['fetchai/fipa:0.1.0','fetchai/default:0.1.0','fetchai/oef:0.1.0']
+ledgers: ['fetchai']
+```
+We must pay attention to the shared_classes and the strategy’s variables. Here we can change the price we would like to buy each reading or the currency we would like to transact with. 
+
+The next file we have to update is the aea-config.yaml file. You can locate this file under your agent’s folder. 
+We are going to modify this file later on before we run the aea but for now, make sure it matches the following code : 
+
+```yaml
+
+aea_version: 0.1.17
+agent_name: m_client
+author: author
+connections:
+- fetchai/stub:0.1.0
+default_connection: fetchai/stub:0.1.0
+default_ledger: fetchai
+description: ''
+fingerprint: ''
+ledger_apis: {}
+license: Apache-2.0
+logging_config:
+ disable_existing_loggers: false
+ version: 1
+private_key_paths: []
+protocols:
+- fetchai/default:0.1.0
+registry_path: ../packages
+skills:
+- author/thermometer_client:0.1.0
+- fetchai/error:0.1.0
+version: 0.1.0
+```
+Important: Do not modify the aea_version. Also if you modified the author makes sure that you changed it for the skills too. 
+For example, if you modify the author to my_author you will have to modify the skills section to :
+
+```yaml
+skills:
+- my_authos/thermometer:0.1.0
+- fetchai/error:0.1.0
+```
+
+## Run the AEAs
+
+In a separate terminal, launch a local OEF node (for search and discovery).
+```bash
+python scripts/oef/launch.py -c ./scripts/oef/launch_config.json
+```
+
+### Fetch.ai ledger payment
+
+Create the private key for the weather client AEA.
+
+```bash
+aea generate-key fetchai
+aea add-key fetchai fet_private_key.txt
+```
+
+### Update the AEA configs
+
+Both in `my_aea/aea-config.yaml` and `my_client/aea-config.yaml`, replace ```ledger_apis```: {} with the following.
+```yaml
+ledger_apis:
+  fetchai:
+    network: testnet
+```
+### Fund the temperature client AEA
+
+Create some wealth for your weather client on the Fetch.ai testnet. (It takes a while).
+
+```bash 
+aea generate-wealth fetchai
+```
+
+Run both AEAs from their respective terminals
+
+```bash 
+aea add connection fetchai/oef:0.1.0
+aea install
+aea run --connections fetchai/oef:0.1.0
+```
+You will see that the AEAs negotiate and then transact using the Fetch.ai testnet.
+
+### Ethereum ledger payment
+
+A demo to run the same scenario but with a true ledger transaction on the Ethereum Ropsten testnet. 
+This demo assumes the temperature client trusts our agent to send the temperature data upon successful payment.
+
+Create the private key for the thermometer client AEA.
+
+```bash
+aea generate-key ethereum
+aea add-key ethereum eth_private_key.txt
+```
+
+### Update the AEA configs
+
+Both in `my_aea/aea-co
```

Comment:
Did you mention somewhere that they need to connect the thermometer etc? Cannot find it atm